re.rizin.cutter.yml

```diff
@@ -0,0 +1,85 @@
+app-id: re.rizin.cutter
+command: cutter
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
```

Comment:
tried this?
```suggestion
runtime-version: 5.15-22.08
```

Reply:
Moving to 22.08 forces me to use the llvm14 sdk but I get errors with the bundled python version when building pyside.

Reply:
it should work.

Reply:
Working on it then I guess

Reply:
Fixed